apps/cookbook/src/app/kirby-christmas/kirby-christmas.component.scss

```
@@ -0,0 +1,201 @@
+.snow-container {
+  overflow-y: hidden;
+  position: relative;
+  height: 100vh;
+  width: 100vw;
+  background-color: #313163;
+}
+
+.kirbylight1 {
+  right: 0;
+  position: absolute;
+  z-index: 1;
+  top: 0;
+}
+
+.title {
+  color: #fff;
+  font-size: 3rem;
+  padding: 10px;
+  text-align: center;
+
+  @media screen and (max-width: 769px) {
+    font-size: 2rem;
+  }
+}
+
+.middle {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  inset: 0;
+  margin: auto;
+  width: 40rem;
+  height: 40rem;
+  @media screen and (max-width: 769px) {
+    width: 20rem;
+    height: 20rem;
+  }
+
+  z-index: 4;
+}
+
+.tree {
+  margin-left: 10%;
+  margin-right: auto;
+  width: 40%;
+  height: 40%;
+  position: absolute;
+  bottom: 0;
+  z-index: 3;
+
+  @media screen and (max-width: 769px) {
+    width: 30%;
+    height: 15%;
+  }
+}
+
+ellipse {
+  animation: glowing 1s infinite 1000ms alternate ease-out;
+}
+
+.kirbylight2 {
+  left: 0;
+  top: 0;
+  position: absolute;
+  z-index: 1;
+}
+
+.kirbybulb1 {
+  animation: glowing 2000ms infinite 1000ms alternate ease-out;
+}
+
+.kirbybulb2 {
+  animation: glowing 2000ms infinite 2000ms alternate ease-out;
+}
+
+.kirbybulb3 {
+  animation: glowing 2000ms infinite 3000ms alternate ease-out;
+}
+
+.kirbybulb4 {
+  animation: glowing 2000ms infinite 4000ms alternate ease-out;
+}
+
+.kirbybulb5 {
+  animation: glowing 2000ms infinite 5000ms alternate ease-out;
+}
+
+.kirbybulb6 {
+  animation: glowing 2000ms infinite 6000ms alternate ease-out;
+}
+
+.kirbybulb7 {
+  animation: glowing 2000ms infinite 7000ms alternate ease-out;
+}
+
+.kirbybulb8 {
+  animation: glowing 2000ms infinite 8000ms alternate ease-out;
+}
+
+.kirbybulb9 {
+  animation: glowing 2000ms infinite 9000ms alternate ease-out;
+}
+
+.kirbybulb10 {
+  animation: glowing 2000ms infinite 10000ms alternate ease-out;
+}
```

Instead of creating 10 almost identical classes manually, you can do something like this instead:

```suggestion
@for $i from 1 to 10 {
  .kirbybulb#{$i} {
    animation: glowing 2000ms infinite #{$i * 1000}ms alternate ease-out;
  }
}
```

---

Thanks for your suggestion! That's way better :D